microsoft-edge/toc.yml

```diff
@@ -459,12 +459,22 @@
               href: webview2/get-started/wpf.md
         - name: Sample code
           href: webview2/code-samples-links.md
-        - name: Concepts
+        - name: Concepts and How-To
           items:
+            - name: Automate with Edge Driver
```

Comment:
I don't think alphabetical ordering is correct; it's random ordering.  I think @jasonstephen15 should arrange these nodes in a flat, logically grouped sequence to put related articles close together.

Note about nesting/depth: If we discover a group of 4 highly related articles, _possibly_ create a topical parent node to group them - though I'm told that nesting should be avoided, and it's actually good (for accessibility & usability) to have more than 7 siblings in a node (possibly many more).

Reply:
We'll address grouping in v2.